config/initializers/services.rb

```diff
@@ -32,3 +32,6 @@ class ServiceNotRegisteredException < Exception; end
   :panopticon,
   GdsApi::Panopticon.new(Plek.new.find('panopticon'),
                          bearer_token: ENV['PANOPTICON_BEARER_TOKEN'] || 'example'))
+
+require 'gds_api/rummager'
+CollectionsPublisher.services(:rummager, GdsApi::Rummager.new(Plek.new.find('search')))
```

Comment:
The plek call should be `Plek.new.find('rummager')`.  When the app was renamed, the search vhost was left in place to avoid breaking everything, but should be considered deprecated.